packages/grid/test/filtering.test.js

```diff
@@ -206,8 +206,11 @@ describe('filtering', () => {
 
     grid._filters[0].value = '99';
     flushFilters(grid);
+
+    grid.allRowsVisible = true;
+    await nextFrame();
     flushGrid(grid);
-    expect(grid.$.items.querySelectorAll('tr:not([hidden])')).to.have.length(18);
+    expect(grid.$.items.querySelectorAll('tr:not([hidden])')).to.have.length(19);
```

Comment:
This old test had an invalid assertion. After filtering with the text "99", there are actually 19 items remaining in the item set.